clang/lib/CIR/Lowering/ThroughMLIR/LowerCIRToMLIR.cpp

```diff
@@ -965,6 +973,27 @@ class CIRYieldOpLowering
   }
 };
 
+class CIRConditionOpLowering
```

Comment:
It seems ConditionOp only be used for loops. Should we move to LowerCIRLoopToSCF.cpp?

Reply:
I move the CIRConditionOpLowering to LoweriCIRLoopToSCF